### PR TITLE
minor: Wrapped chained method calls

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
Long method chains are hard to read, wrapped them onto a new line. Prerequisite of https://github.com/checkstyle/checkstyle/pull/11231
Suggested in - https://github.com/checkstyle/checkstyle/pull/11231#discussion_r829070035

### CODE REVIEW

**Review by nick-mancuso — APPROVED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java [hunk 1]
```diff
@@ -727,7 +727,8 @@ private static void validatePropertySection(String fileName, String sectionName,
 
         assertWithMessage(
                 fileName + " section '" + sectionName + "' should show properties: " + properties)
-                        .that(properties).isEmpty();
+                        .that(properties)
```
Comment (pbludov) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java:730]: I think that the wrapping on the line #1422 is more readable.
Such indent looks like it is bound to the string message.

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java [hunk 2]
```diff
@@ -1595,7 +1597,8 @@ public void testAllStyleRules() throws Exception {
 
             assertWithMessage(
                     fileName + " requires the following check(s) to appear: " + styleChecks)
-                            .that(styleChecks).isEmpty();
+                            .that(styleChecks)
+                            .isEmpty();
```
Comment (pbludov) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java:1601]: same here

**Review by pbludov — APPROVED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java (hunk 1 above)
Comment (Vyom-Yadav) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java (hunk 2 above)
Comment (Vyom-Yadav) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java]: Done.

**Review by romani — APPROVED** [review]
This idea of placing each call on separate line might be ok in general for test code. But in main code there will problems, as it extends method size significantly.
Forcing people to refactor method split is not good.

**Comment by Vyom-Yadav** [comment]
> This idea of placing each call on separate line might be ok in general for test code. But in main code there will problems, as it extends method size significantly.
Forcing people to refactor method split is not good.

It can be configured to suit anyone's needs.

**Pull request merged**